.travis.yml

```diff
@@ -34,12 +37,13 @@ before_install:
   - rustup component add rustfmt-preview
```

Comment:
On line 33, update the LD_LIBRARY_PATH to include /usr/local/lib/x86_64-linux-gnu so that libopencv_x can be found.

```
  - export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/local/lib/x86_64-linux-gnu
```

P.S. I am not sure why this wasn't an issue before...

Reply:
Hmm, probably because of files copying. I think we need to avoid it whenever possible, because it's confusing build system and can lead to improper configuration: e.g. you reference ones libraries while having it in another place, but they are copied from one location to another so it works on your environment, but suddenly stops working on someone else's environment.

Reply:
I think when linking, the build system should be able to search recursively within the folder `/usr/local/lib`. So it should work. But I don't know why it doesn't. So my fix for now is to add `/usr/local/lib/x86_64-linux-gnu` directly.

Reply:
Souns weird because this test is actually unchanged...